shell/platform/glfw/flutter_glfw.cc

```diff
@@ -149,10 +149,57 @@ struct FlutterDesktopPluginRegistrar {
 
 // State associated with the messenger used to communicate with the engine.
 struct FlutterDesktopMessenger {
+  FlutterDesktopMessenger() = default;
+
+  void AddRef() { ref_count_.fetch_add(1); }
+
+  void Release() {
+    int32_t old_count = ref_count_.fetch_sub(1);
+    if (old_count <= 1) {
+      delete this;
+    }
+  }
+
+  FlutterDesktopEngineState* GetEngine() const { return engine_; }
+  void SetEngine(FlutterDesktopEngineState* engine) {
+    std::scoped_lock lock(mutex_);
+    engine_ = engine;
+  }
+
+  std::mutex& GetMutex() { return mutex_; }
+
+ private:
+  FML_DISALLOW_COPY_AND_ASSIGN(FlutterDesktopMessenger);
```

Comment:
We don't use these macros in the desktop embeddings, as they are newer than the C++ style guide change that banned them. Copy and assign should be removed by deleting them: https://google.github.io/styleguide/cppguide.html#Copyable_Movable_Types

Reply:
done